examples/audio_classifier/raspberry_pi/classify.py

```diff
@@ -0,0 +1,133 @@
+    score_threshold: The score threshold of classification results.
+    overlapping_factor: Target overlapping between adjacent inferences.
+  """
+
+  if (overlapping_factor <= 0) or (overlapping_factor >= 1.0):
+    raise ValueError('Overlapping factor must be between 0 and 1.')
+
+  if (score_threshold < 0) or (score_threshold > 1.0):
+    raise ValueError('Score threshold must be between (inclusive) 0 and 1.')
+
+  # Initialize a plotter instance to display the classification results.
+  plotter = Plotter()
+
+  classification_result_list = []
+
+  def save_result(result: audio.AudioClassifierResult, timestamp_ms: int):
+    result.timestamp_ms = timestamp_ms
+    classification_result_list.append(result)
+
+  # Initialize the audio classification model.
+  base_options = python.BaseOptions(model_asset_path=model)
+  options = audio.AudioClassifierOptions(
+      base_options=base_options, running_mode=audio.RunningMode.AUDIO_STREAM,
+      max_results=max_results, score_threshold=score_threshold,
+      result_callback=save_result)
+  classifier = audio.AudioClassifier.create_from_options(options)
+
+  # Initialize the audio recorder and a tensor to store the audio input.
+  # The sample rate may need to be changed to match your input device.
+  # For example, an AT2020 requires sample_rate 44100.
+  buffer_size, sample_rate, num_channels = 15600, 16000, 1
+  audio_format = containers.AudioDataFormat(num_channels, sample_rate)
+  record = audio_record.AudioRecord(num_channels, sample_rate, buffer_size)
+  audio_data = containers.AudioData(buffer_size, audio_format)
+
+  # We'll try to run inference every interval_between_inference seconds.
+  # This is usually half of the model's input length to create an overlapping
+  # between incoming audio segments to improve classification accuracy.
+  input_length_in_second = float(len(
+      audio_data.buffer)) / audio_data.audio_format.sample_rate
+  interval_between_inference = input_length_in_second * (1 - overlapping_factor)
+  pause_time = interval_between_inference * 0.1
+  last_inference_time = time.time()
+
+  # Start audio recording in the background.
+  record.start_recording()
+
+  # Loop until the user close the classification results plot.
+  while True:
+    # Wait until at least interval_between_inference seconds has passed since
+    # the last inference.
+    now = time.time()
+    diff = now - last_inference_time
+    if diff < interval_between_inference:
+      time.sleep(pause_time)
+      continue
+    last_inference_time = now
+
+    # Load the input audio from the AudioRecord instance and run classify.
+    data = record.read(buffer_size)
+    # audio_data.load_from_array(data.astype(np.float32))
+    audio_data.load_from_array(data)
+    classifier.classify_async(audio_data, time.time_ns() // 1_000_000)
+
+    # # Plot the classification results.
+    if classification_result_list:
+      plotter.plot(classification_result_list[0])
```

Comment:
Are we also printing the results to the terminal? If not, we should - plus mention the monitor option being available in the readme like we do with image classification and object detection.

Reply:
I removed them for the Raspberry Pi samples to make the visualization run smoother.

Reply:
I'll add them commented out for now so it can be uncommented out for debugging reasons.